README.md

```diff
@@ -1,6 +1,50 @@
 [![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
 
-# docs-website
+# New Relic Docs
+
+This repo contains the documentation website code and Markdown source files for  our preview Docs site: docs-preview.newrelic.com. When it's ready, all of our docs will live here.
+
+Read on to learn more about who we are and how you can contribute.
+
+## We'd like your help
+
+From early on, our Docs site has been managed by a team of writers. We've always welcomed contributions from anyone at New Relic. We credit our technical accuracy and comprehensive documentation to this openness. In a sense, it's documentation as conversation.
+
+Open sourcing our docs seemed like a great way to expand this conversation beyond New Relic.
+
+We're here working every day to improve our docs, but we'd love to hear from you. Come join the conversation.
+
+## Get started
+
+On every one of our docs, you can [create an issue]([link to issue template page](https://github.com/newrelic/docs-website/issues/new/choose)) or [edit a page]([link to more detailed instructions about contributing](https://github.com/newrelic/docs-website/blob/develop/CONTRIBUTING.md)). 
+
+### Create an issue
+
+Issues are a quick way to give us feedback about our docs. We'll review your issue and follow up with you if we have any questions.
+
+We have a set of issue templates to help you get started. You can create an issue to let us know when you find an error or notice something missing. You can also let us know about things you'd like to see.
+
+### Edit a page
+
+If you'd like to get more directly involved, you can make some changes yourself. On any article, click the **Edit this page** button to get started making a change.
+
+When you edit a doc, you'll be able to start editing it on GitHub right away. See our [Contributors guide](https://github.com/newrelic/docs-website/blob/readme-update/CONTRIBUTING.md) for information on how to fork our site and submit pull requests. This includes information on what you'll need to build the site locally on your computer.
+
+Our [Style guide](https://github.com/newrelic/docs-website/blob/readme-update/STYLE_GUIDE.md) will give you some insight into how we think about writing and documenation, as well as our flavor of Markdown.
```

Comment:
I'd add an explicit line here that indicates they don't need to read the style guide. Something like:

> Reading the style guide is totally optional! Our writers are here to ensure things are formatted and worded right—what we're looking for most from contributors is your technical insight.

Reply:
+1 - again to avoid making it sound more complicated and time consuming than it has to be.